### DISCUSSION TITLE
Official limit for number of solution

### DISCUSSION THREAD

**Discussion by neinecke** [esa/SpOC2#23] [Quantum Constellations Q&A] [answered]
Dear SpOC organizer,

I have a short question. What is the officially allowed number of solutions. The website description says 100 but when uploading results we get the following error:
[quantum-communications-constellations] The decision vector may contain at most 50 solutions.

Best,
Nils

**Comment by EmBlazquez** [comment, marked as answer]
Hi @neinecke ,

I have updated optimize to allow for submissions up to 100 solutions, you should now be able to submit :)

Best, 

Emmanuel

**Reply by neinecke** [reply]
Thanks a lot for the quick fix Emmanuel.

**Comment by dietmarwo** [comment]
Oops, good that I noticed this change here, since this influences the score. Have to reupload then. Hopefully all participants see this change. For me the score changes from -6393 to -6399 just for going from 50 to 100 solutions. The "real" value for the whole pareto front is -6403, so the difference is more than halved. Currently only the top 2 teams have adapted, all other uploads have used only 50 solutions since they are all from April. 

I got two times

"This file does not follow the correct file format. Read about the expected file format in the expansion panels on the submit page."

then I reuploaded the same file and got: 

[quantum-communications-constellations] You have already submitted this solution on 2023-06-16 12:53 (score: -6399.112568780042).

Finally the new score is shown and the upload succeeded - despite the error message. 
But after the first upload of the same file my score in the Leaderboard was unaffected.

Its all ok now, but may be you have logs to check what was going on. From monday most will be busy with GTOC12 anyway.
